.github/problem-matchers/depmod.json

```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+        {
+            "__comment_owner": "match depmod warnings/errors",
+            "owner": "depmod",
+            "pattern": [
+                {
+                    "__comment_regexp1": "depmod: ERROR: Cycle detected: crypto -> crypto_algapi -> crypto",
+                    "__comment_regexp2": "depmod: ERROR: Found 2 modules in dependency cycles!",
+                    "regexp": "^depmod:\\s+(FATAL|ERROR|WARNING|NOTICE|INFO|DEBUG):\\s+(.*)$",
```

Comment:
Also, given how roughly generic this could be if `^depmod:` were `^[^:]+:` should there be a catch-all problem matcher for "unknown" tools that follow this (common) convention?

Reply:
Yes, I think that is worthwhile so I will go ahead and change `depmod.json` to `generic.json`.